src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -215,7 +217,8 @@ export class TemplateEditorComponent extends UnsubscribeOnDestroyAdapter impleme
         this.existingTemplate = true;
         this.statusMessage = 'Template saved';
       },
-      () => {
+      (error) => {
+        console.log(`Save failed: $(error)`);
```

Comment:
I would use `console.error` to have the stack trace and more information displayed in the console.

Reply:
Good suggestion.  Made the chagne.